tasks.py

```diff
@@ -0,0 +1,136 @@
+"""
+PyInvoke developer task file
+https://www.pyinvoke.org/
+
+These tasks can be run using `invoke <NAME>` or `inv <NAME>` from the project root.
+
+To show all available tasks `invoke --list`
+
+To show task help page `invoke <NAME> --help`
+"""
+import invoke
+
+from scripts import check_type_hint_coverage
+
+_CHECK_HELP_DESC = "Only checks for needed changes without writing back. Exit with error code if changes needed."
+_EXCLUDE_HELP_DESC = "Exclude files or directories"
+
+
+@invoke.task(
+    help={
+        "check": _CHECK_HELP_DESC,
+        "exclude": _EXCLUDE_HELP_DESC,
+    }
+)
+def sort(ctx, path=".", check=False, exclude=None):
+    """Sort module imports."""
+    cmds = ["isort", path, "--profile", "black"]
+    if check:
+        cmds.append("--check-only")
+    if exclude:
+        cmds.extend(["--skip", exclude])
+    ctx.run(" ".join(cmds))
+
+
+@invoke.task(
+    help={
+        "check": _CHECK_HELP_DESC,
+        "exclude": _EXCLUDE_HELP_DESC,
+        "sort": "Disable import sorting. Runs by default.",
+    }
+)
+def fmt(ctx, path=".", sort_=True, check=False, exclude=None):
```

Comment:
Wanted to call this out, as opinions here my vary.
By default I have the `invoke fmt` command also run the isort unless you explicitly tell it not to.

opt-out vs opt-in.

I'm open to either.

Reply:
I like it! Easy enough to modify if we want to later on